augur/export_v2.py

```diff
@@ -939,6 +917,51 @@ def set_description(data_json, cmd_line_description_file):
     except FileNotFoundError:
         fatal("Provided desciption file {} does not exist".format(cmd_line_description_file))
 
+def create_branch_mutations(branch_attrs, node_data):
+    for node_name, node_info in node_data['nodes'].items():
+        if node_name not in branch_attrs:
+            continue # strain name not in the tree
+        if "aa_muts" not in node_info and "muts" not in node_info:
+            continue
+        branch_attrs[node_name]['mutations'] = {}
+        if "muts" in node_info and len(node_info["muts"]):
+            branch_attrs[node_name]["mutations"]["nuc"] = node_info["muts"]
+        if "aa_muts" in node_info:
+            aa = {gene:data for gene, data in node_info["aa_muts"].items() if len(data)}
+            branch_attrs[node_name]["mutations"].update(aa)
+
+def create_branch_labels(branch_attrs, node_data, branch_data):
+    ## start by creating the 'aa' branch label, summarising any amino acid mutations.
+    ## (We have already set mutations on 'branch_attrs' if they exist, just not the label)
```

Comment:
I'm a little confused by this choice, why not just set the label at the same time as the mutations?

Reply:
I felt it was cleaner to separate out the creation of mutations from the creation of labels, but it's a judgement call... do you feel that this is less readable than (a) setting mutations + a label and then (b) setting other labels?

Reply:
Ah, I guess the separation of concerns is slightly different in my head of (a) augur export added mutations/labels and then (b) user specified labels. But looking at this again, it makes sense to leave it as is so that all label creation is in the same place.